apps/dalmatiner_db/src/dalmatiner_tcp.erl

```diff
@@ -45,10 +45,6 @@ loop(Socket, Transport, State) ->
                     {ok, Bs} = metric:list(),
                     Transport:send(Socket, dproto_tcp:encode_metrics(Bs)),
                     loop(Socket, Transport, State);
-%%                {resolution, Bucket} ->
-%%                    Resolution = dalmatiner_opt:resolution(Bucket),
-%%                    Transport:send(Socket, <<Resolution:64/integer>>),
-%%                    loop(Socket, Transport, State);
                 {list, Bucket} ->
                     {ok, Ms} = metric:list(Bucket),
                     Transport:send(Socket, dproto_tcp:encode_metrics(Ms)),
```

Comment:
Remove the dynamic function call on line 50. Only modules that define callbacks should make dynamic calls.